elements/pfe-jump-links/demo/index.html

```diff
@@ -67,7 +67,7 @@ <h1><code>pfe-jump-links</code></h1>
   <h2 style="text-align: center; margin: 60px auto;">PFE Jump Links - standard</h2>
   <div style="margin: 0 auto 400px;grid-column-gap: 30px;max-width:1140px;" class="pfe-l-grid">
     <div class="sticky pfe-l-grid__item pfe-m-12-col pfe-m-3-col-on-lg">
-      <pfe-jump-links-nav id="jumplinks1" sr-text="Page navigation">
+      <pfe-jump-links-nav id="jumplinks1" sr-text="Page navigation" offset="0">
```

Comment:
Instead of coding this as an attribute, could you set it as a custom property? There’s a built in var function in pfelement you can make use of. `this.var(“offset”, height)`. This is nice because then we can use it in CSS calc functions and JS if needed.

Reply:
Making this change now.